src/algorithms/expected_consensus/_index.md

```diff
@@ -2,119 +2,79 @@
-`target = activePowerInSector/networkPower * EC.ExpectedLeadersPerEpoch * numSectorsMiner / numSectorsSampled`
-
-The target ensures that the miner can express the power across all of their sectors through the tickets they have sampled. Specifically, on expectation, checking `numSectorsSampled` (with `numSectorsSampled = ceil(len(provingSet) * EPoStSampleRate)`) challenge tickets in every epoch, a miner will find `minerPower/networkPower * EC.ExpectedLeadersPerEpoch` winning tickets per epoch on expectation. Note that while the sectorPower may differ based on the challenged sector, i.e. in any given epoch a miner may have an advantage (if picking sectors with more than the average across all their sectors) or a disadvantage (conversely) in their election; but over multiple epochs, on expectation the election will be fair.
-
-```text
-We elaborate on the above claim:
+At a high-level, leader election works as follows:
 
-We want the miner's expected wins over time to be equal to w = minerPower/networkPower * EC.ExpectedLeadersPerEpoch
+- A miner draws an appropriate random seed for this epoch
+- They generate an `ElectionProof` from this seed by using a VRF to generate a signature over the seed at a given epoch, as defined in {{<sref randomness>}}. The `ElectionProof` is the VRF Proof.
+  - If the `ElectionProof`'s normalized digest value (i.e. the VRF Digest) is below a miner-specific `ElectionTarget` determined by the miner's power in SPC, it is valid: the miner can craft a block (see {{<sref block_producer>}}). 
+    - The `ElectionProof` Digest must be normalized according to both the `ExpectedLeadersPerEpoch` in EC (a network parameter) and the maximum value of the digest (based on the hash used to produce it).
+    - The `ElectionTarget` is set as the proportion of the miner's `QualityAdjustedPower` over the total network power. Accordingly leader election in EC is proportional to miner power.
+  - Otherwise, the miner tries again in the next epoch.
 
-Take P to be the average miner power over the N sectors in their ProvingSet.
-We have P = SUM_{i=0}^N P_i / N
-
-The miner's likelihood of winning an election in any epoch is, for C tickets randomly drawn
-W = C * target = C * P_i/networkPower * EC.ExpectedLeadersPerEpoch * N/C = N * P_i/networkPower * EC.ExpectedLeadersPerEpoch
-
-with N * P_i ~= minerPower on expectation over enough epochs. That is to say: W = minerPower/networkPower * EC.ExpectedLeadersPerEpoch as wanted.
-```
+Conceptually, EC yields winners proportionally to their power since it enables each miner to generate a single `ElectionProof` whose digest can be normalized to yield a uniformly-drawn random number between 0 and 1. It is then compared to the miner's power in proportion to total network power (i.e. also between 0 and 1). The more powerful the miner, the more frequently their `ElectionProof`s will be valid.
```

Comment:
ditto

Reply:
sorry, what is ditto in reference to?

Reply:
I believe it's total network power --> Total Quality Adjusted Power, ty!